lib/administrate/fields/base.rb

```diff
@@ -16,6 +16,14 @@ def self.searchable?
         false
       end
 
+      def self.search_predicate(attribute, term)
+        "lower(#{attribute}) LIKE ?"
+      end
+
+      def self.search_term(attribute, term)
```

Comment:
Unused method argument - `attribute`. If it's necessary, use `_` or `_attribute` as an argument name to indicate that it won't be used.